### DISCUSSION TITLE
Doesn't Sync with Salesforce

### DISCUSSION THREAD

**Discussion by YG202** [Authenticator-Extension/Authenticator#825] [Q&A]
### Describe the issue

I am trying to use the Google Authenticator chrome extension for my MFA Authenticator with Salesforce, I am able to scan the QR code that salesforce provides but salesforce requires me to enter a verification code and when I enter the code given to me by the Authenticator I get the error "Invalid verification code".

### Browser

Chrome

### Browser Version

Most recent I think

### Extension Version

6.3.2

**Comment by mymindstorm** [comment]
Try going to Menu -> Sync clock. If that doesn't work, use the QR button at the top right and scan using a phone; if that does not work then it is a Salesforce issue.
